trlx/data/configs.py

```diff
@@ -60,6 +56,30 @@ def from_dict(cls, config: Dict[str, Any]):
         return cls(**config)
 
 
+@dataclass
+class TokenizerConfig:
+    """
+    Config for a model.
+
+    :param tokenizer_path: Path or name of the tokenizer (local or on huggingface hub)
+    :type tokenizer_path: str
+
+    :param padding_side: Padding side
+    :type padding_path: str
+
+    :param truncation_side: Truncation side
+    :type truncation_side: str
+    """
+
+    tokenizer_path: str
```

Comment:
(Thinking out loud on usability): I feel like we could make this optional and default to using the `model_path` to load the tokenizer out of `AutoTokenizer.from_pretrained` in the base trainer)

Reply:
Also the `tokenizer_` part seems redundant being already in a `TokenizerConfig `